userguide/snippets/x_drivetrays.rst

```diff
@@ -12,9 +12,9 @@ trays must be placed in the empty bays.
 A standard drive tray installation order simplifies support and is
 strongly recommended:
 
-* SSD drives for SLOG, if present
+* SSD drives for W, if present
```

Comment:
This needs to explain what the W stands for:
`` SSD drives for write cache (:literal:`W`), if present ``
`` SSD drives for read cache (:literal:`R`), if present ``